the-solutions-journal.csl

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>The Solutions Journal</title>
+    <id>http://www.zotero.org/styles/the-solutions-journal</id>
+    <link rel="self" href="http://www.zotero.org/styles/the-solutions-journal"/>
+    <link rel="template" href="http://www.zotero.org/styles/nature"/>
+    <link href="http://www.thesolutionsjournal.com/node/457" rel="documentation"/>
+    <author>
+      <name>Jafet Andersson</name>
+      <uri>http://www.smhi.se/en/research/research-departments/hydrology/jafet-andersson-1.18716</uri>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="science"/>
+    <category field="generic-base"/>
```

Comment:
please remove this -- we're only using this for general purpose style guides, not specific journals.